### ISSUE TITLE
Expecting a finite number

### ISSUE THREAD

**Issue by Lisennk** [sindresorhus/crypto-random-string#15]
Given example
```js
const cryptoRandomString = require('crypto-random-string');
cryptoRandomString({length: 10});
```
throws TypeError: expecting a finite number

**Comment by sindresorhus** [comment]
The test is passing: https://github.com/sindresorhus/crypto-random-string/blob/9cc193ba09acd9df65509913c68a91d1c7f6006d/test.js#L19

**Comment by khushalEnuke** [comment]
Same issue on Live environment. 

`const cryptoRandomString = require('crypto-random-string');
let uniqueId = cryptoRandomString({ length: 7 });
`

**Comment by elishaterada** [comment]
We ran into this issue, `npm audit fix` fixed the problem. I'm guessing that the child dependencies of this package have had some issues.

**Comment by sindresorhus** [comment]
> I'm guessing that the child dependencies of this package have had some issues.

This package doesn't have any actual dependencies: https://github.com/sindresorhus/crypto-random-string/blob/9cc193ba09acd9df65509913c68a91d1c7f6006d/index.js#L1-L3

**Comment by williamknn** [comment]
Same issue here. When the application starts, the length value is `undefined`. A simple `return` if is `undefined` would solve the problem.

**Comment by andresilva-cc** [comment]
This error is happening to me. The strange thing is that in my development machine (running Debian 10) it works, but on the production server (running CentOS 7) it doesn't.

**Comment by philippbosch** [comment]
I ran into the exact same problem and figured out what happened. Maybe this helps others.

Before I found this package, I initially installed the [unique-string](https://github.com/sindresorhus/unique-string) package which has a dependency on `"crypto-random-string": "^2.0.0"`. When installing `crypto-random-string` npm apparently just used the already installed version instead of the latest available. So I ended up with version 2.0.0 for which the function signature is different, i.e. instead of passing something like `{ length: 10 }` it expects just the length as an integer as the first argument. So when you pass it an object, it will fail with `TypeError: Expected a finite number`.

I fixed it by uninstalling both packages and re-instaling `crypto-random-string`. This gave me version 3.3.0.
